aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -220,6 +291,9 @@ sub create_vm_images
         }
         if ($newimage) {
             $main::this_app->info("Created new VM image ID: ", $newimage->id);
+            if (ref $permissions eq ref {}) {
```

Comment:
@stdweird It's strange in that case I get this error:
`Can't use string ("1") as a HASH ref while "strict refs" in use at /home/lsimngar/UGent/hpcugent/aii/aii-opennebula/target/lib/perl/AII/opennebula.pm`

from:
`$chown{uid} = defined($permissions->{owner}) ? $permissions->{owner} : -1;`

That is the reason why I have included this condition to be sure that I'm getting a HASH, maybe something is wrong in the code, I have to check....